test/test_operations.py

```diff
@@ -1571,6 +1571,17 @@ def test_fn(*indices):
         for dtype in (torch.long, torch.int32, torch.bool)
     ], test_fn)
 
+  def test_conv2d_backward(self):
+    # Somehow eager cpu produces different results than us, and
```

Comment:
@JackCaoG do you know why?

Reply:
it is not that uncommon that we give slightly different result, is it off by a lot? If is possible that result is different because we run some optimization pass which make code runs faster but not as accurate.